nemo_rl/models/policy/megatron_policy_worker.py

```
@@ -24,6 +24,67 @@
+                        f.write(new_content)
+                    print("Successfully patched transformer_engine.")
+                except OSError as e:
+                    print(
+                        f"Could not write patch to transformer_engine (permission denied?): {e}"
+                    )
+            else:
+                print("Could not find patch locations in permutation.py")
+
+    except (ImportError, FileNotFoundError, AttributeError):
+        pass
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)